external/fv3fit/fv3fit/_shared/stacking.py

```diff
@@ -113,3 +113,27 @@ def _get_chunk_indices(chunks):
         indices.append(list(range(start, start + chunk)))
         start += chunk
     return indices
+
+
+def infer_dimension_order(ds: xr.Dataset) -> Tuple:
```

Comment:
These are copied from their previous location in `predictor.py` and made public for use in models' `.predict` methods.